### PR TITLE
Add 11.22 release notes

### PR DESCRIPTION
**Pull request by tas50**
#### Description

New week. New release

#### Types of changes

- [ ] Functional documentation bug fix (i.e., broken link or some other busted behavior)
- [ ] New functional doc capabilities (i.e., filter search results)
- [x] New content
- [ ] Revision to existing content
- [x] Chore (non-breaking change that does not add functionality or fix an issue)

#### Checklist

- [x] I have read the **README** document about contributing to this repo.
- [x] I have tested my changes locally and there are no issues.
- [x] All commits are signed.

### CODE REVIEW

File: releases/2024-09-17-mondoo-11.22-is-out.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+## 🥳 Mondoo 11.22 is out! This release includes FOO, BAR, BAZ, and more!
+
+Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)
+
+---
+
+## 🎉 NEW FEATURES
+
+### Cases Cases Cases!
```
Comment (misterpantz) [inline releases/2024-09-17-mondoo-11.22-is-out.md:18]:
```suggestion
### Cases, Cases, Cases!
```

Comment (misterpantz) [reply releases/2024-09-17-mondoo-11.22-is-out.md:18]: If that's the heading @charlesjohnson wants. :)

File: releases/2024-09-17-mondoo-11.22-is-out.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+## 🧹 IMPROVEMENTS
+
+### New checks in the CIS Azure Foundations benchmark policy
+
+Harden your Microsoft Azure subscriptions with expanded checks in the CIS Azure Foundations Benchmark policy. This policy now ensures that Azure Key Vault rotation is configured and AppService HTTP logs are enabled.
+
+## 🐛 BUG FIXES AND UPDATES
+
+- Improve rendering of complex remediation text in CIS policies.
+- Fix the alignment of Start Scanning and Browse Integration buttons in new spaces.
```
Comment (misterpantz) [inline releases/2024-09-17-mondoo-11.22-is-out.md:35]:
```suggestion
- Fix the alignment of the Start Scanning and Browse Integration buttons in new spaces.
```

File: releases/2024-09-17-mondoo-11.22-is-out.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+- Improve handling of SMBIOS data on Windows to prevent errors displaying in the asset overview.
+- Fix failures to scan more than 100 S3 buckets at a time.
+- Resolve an AWS Serverless integration error shown in the console integration details page.
```
Comment (misterpantz) [inline releases/2024-09-17-mondoo-11.22-is-out.md:38]:
```suggestion
- Resolve an AWS serverless integration error shown in the console integration details page.
```

File: releases/2024-09-17-mondoo-11.22-is-out.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+- Improve the rendering of some CVE descriptions.
+- Update AWS policies to use the latest RDS MQL resources.
+- Install cnspec from install.mondoo.com instead of mondoo.com during Azure VM scans.
+- Fix false negatives when checking bootloader config on RHEL 9 based Linux distributions.
```
Comment (misterpantz) [inline releases/2024-09-17-mondoo-11.22-is-out.md:42]:
```suggestion
- Fix false negatives when checking bootloader config on Linux distributions based on RHEL 9.
```

File: releases/2024-09-17-mondoo-11.22-is-out.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+- Skip GDM and firewalld checks on RHEL 9 based Linux distributions when the packages aren't installed.
```
Comment (misterpantz) [inline releases/2024-09-17-mondoo-11.22-is-out.md:43]:
```suggestion
- Skip GDM and firewalld checks when those packages aren't installed on Linux distributions based on RHEL 9.
```

File: .github/actions/spelling/expect.txt [hunk 6]
```diff
@@ -276,6 +276,7 @@ Qwm
 ratebasedstatement
 regexmatchstatement
 regexpatternsetreferencestatement
+remediates
```
Comment (tas50) [inline .github/actions/spelling/expect.txt:279]:
```suggestion
```

I moved this to the main dictionary file

**Review by misterpantz — APPROVED** [review]

**Pull request merged**